README.md

```diff
@@ -119,6 +119,15 @@ Serve files relative to `path`.
 Byte offset at which the stream starts, defaults to 0. The start is inclusive,
 meaning `start: 2` will include the 3rd byte in the stream.
 
+##### caseSensitive
+
+When set to `true` all the requests against the file system are made case sensitive even on case insensitive file systems.
+This effects not only the file name but the whole path.
+When set to `false` the default behavior of the filesystem is used.
```

Comment:
Passing in `caseSensitive: false` does have a weird feel to the API... it feels like it should thus *not* be case-sensitive, vs depend on the file system. I'm trying to think of a good API name for this functionality, and don't have a good one.

Would it be possible to maybe even make `caseSensitive: false` work? For example, to have this option be `true`, or `false, or `'fs'`? If not, as I cannot think of a good API name that would make sense as true/false, maybe use a string like `'fs'` instead of accepting `false`?